src/python/WMComponent/RucioInjector/Database/MySQL/GetCompletedBlocks.py

```diff
@@ -86,21 +85,29 @@ def format(self, result):
         # NOTE: We need to rename all the keys to follow the cammelCase standard. And also to comply
```

Comment:
If we decide to return a dictionary instead of a list of records (for which I am not 100% sure yet), all that needs to be reflected in the doc string as well.

Reply:
Agreed. Let's decide what we will do first and then I can make the changes

Reply:
Upon our discussion and tests, decision has been made about that - Lets return a dictionary and simply reflect that change in the DAO's docstring.